lua/.gitignore

```diff
@@ -0,0 +1,19 @@
+# Build output directories
+workdir/
+build/
+
+# Kconfig-generated files
+*.cmd
+*.o
+*.d
+*.confdata
+*.tab.c
+*.tab.h
+
+# Unikraft specific
+fc.*.json
+
+# Editor / OS junk
+*.swp
+*~
+.DS_Store
```

Comment:
Use the `.gitignore` file from [Nginx](https://github.com/unikraft/catalog-core/blob/main/nginx/.gitignore).